src/envoy/tcp/alpn_proxy/config.cc

```diff
@@ -0,0 +1,120 @@
+/* Copyright 2019 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "src/envoy/tcp/alpn_proxy/config.h"
+#include "common/config/filter_json.h"
+#include "envoy/network/connection.h"
+#include "envoy/registry/registry.h"
+#include "src/envoy/tcp/alpn_proxy/alpn_proxy.h"
+#include "src/envoy/utils/config.h"
+
+namespace Envoy {
+namespace Tcp {
+namespace AlpnProxy {
+
+Network::FilterFactoryCb AlpnProxyConfigFactory::createFilterFactory(
```

Comment:
This can be safely removed. JSON config is no longer required.

Reply:
Done